src/schema/v2/partners.ts

```diff
@@ -10,11 +9,18 @@ import {
   GraphQLFieldConfig,
 } from "graphql"
 import { ResolverContext } from "types/graphql"
+import {
+  connectionWithCursorInfo,
+  createPageCursors,
+} from "./fields/pagination"
+import { connectionFromArray } from "graphql-relay"
+import { convertConnectionArgsToGravityArgs } from "lib/helpers"
+import { pageable } from "relay-cursor-paging"
 
 const Partners: GraphQLFieldConfig<void, ResolverContext> = {
-  type: new GraphQLList(Partner.type),
+  type: connectionWithCursorInfo({ nodeType: PartnerType }).connectionType,
```

Comment:
Since the partner category type already references the `Partners` list type, I might leave this as-is and explicitly add a new `PartnersConnection` variable (to be exported) for the connection type.